src/Maestro/Microsoft.DotNet.Maestro.Tasks/Proxies/GetEnvProxy.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Proxies
+{
+    internal abstract class IGetEnvProxy
```

Comment:
Is there a reason to use an abstract class instead of an interface here?

Reply:
Nope, I think I just looked at an NUnit example and used what they used, but in this case an interface should work the same.